### PR TITLE
Added valications to collaboration.model

### PR DESCRIPTION
**Pull request by Timbit87**
I added validations and belongs_to to the collaboration model.

### CODE REVIEW

File: app/models/collaboration.rb [hunk 1]
```diff
@@ -1,2 +1,4 @@
 class Collaboration < ApplicationRecord
+  belongs_to :brand
+  validates :title, :description, :price, :start_date, :closing_date, presence: true
```
Comment (StevenEReyes) [inline app/models/collaboration.rb:3]: Can you please add :tag to validations?

**Review by StevenEReyes — COMMENTED** [review]
Can you please add :tag to validations?

File: test/controllers/brands/collaborations_controller_test.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+require "test_helper"
```
Comment (Gazwai) [inline test/controllers/brands/collaborations_controller_test.rb:1]: I think you could remove these test files 🤔

Comment (Timbit87) [reply test/controllers/brands/collaborations_controller_test.rb:1]: The whole test folder?

Comment (Gazwai) [reply test/controllers/brands/collaborations_controller_test.rb:1]: @Timbit87 ah they seem to be auto generated. You can leave them in as they wont cause any problems. Sorry for the confusion and probably just a nitpicky comment 🙏

**Pull request merged**